feature/bookmarks/src/main/java/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksScreen.kt

```diff
@@ -61,8 +70,29 @@ internal fun BookmarksRoute(
     )
 }
 
+/**
+ * Displays the user's bookmarked articles. Includes support for loading and empty states.
+ */
+@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+@Composable
+internal fun BookmarksScreen(
+    feedState: NewsFeedUiState,
+    removeFromBookmarks: (String) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    when (feedState) {
+        Loading -> LoadingState(modifier)
+        is Success ->
+            if (feedState.feed.isNotEmpty()) {
+                BookmarksGrid(feedState, removeFromBookmarks, modifier)
+            } else {
+                EmptyState(modifier)
```

Comment:
Looks awesome now, thanks! 

However, we are pretty sure there will be no other situation except `isNotEmpty()` in the future, so why not removing the curly bracket to make it even more clear? 

This way, we will be able to put the `if` condition on `Success` body without using the curly bracket, as well as removing it from the last line and the result will be:

```
when (feedState) {
        Loading -> LoadingState(modifier)
        is Success -> if (feedState.feed.isNotEmpty()) {
            BookmarksGrid(feedState, removeFromBookmarks, modifier)
        } else EmptyState(modifier)
    }

```

Please let me know what you think :)

Reply:
Hmm I'm not really a big fan of if/else statements without curly brackets unless they're really trivial and you can fit the whole thing on one line. So in this case I'd opt for the version with the curly brackets, to be most explicit. But I don't have a super strong preference either way.

Reply:
Alright then, I just removed the space before the `if` condition.